cmake/nuttx/px4_impl_nuttx.cmake

```diff
@@ -248,43 +248,48 @@ function(px4_nuttx_add_export)
 		REQUIRED OUT CONFIG THREADS
 		ARGN ${ARGN})
 
-	set(nuttx_src ${PX4_BINARY_DIR}/${CONFIG}/NuttX)
+	set(nuttx_build_src ${PX4_BINARY_DIR}/${CONFIG}/NuttX)
+	set(nuttx_export_dir ${nuttx_build_src}/nuttx/nuttx-export)
 
-	# all patches
+	# nuttx cmake dependency files
+	set(nuttx_copy ${PX4_BINARY_DIR}/${CONFIG}_nuttx_copy.stamp)
+	set(nuttx_configure ${PX4_BINARY_DIR}/${CONFIG}_nuttx_configure.stamp)
+	set(nuttx_export ${PX4_BINARY_DIR}/${CONFIG}_nuttx_export.stamp)
+
+	# all nuttx patches
 	file(GLOB nuttx_patches ${PX4_SOURCE_DIR}/nuttx-patches/*.patch)
 	list(SORT nuttx_patches)
 
 	# copy
-	file(GLOB_RECURSE nuttx_all_files ${PX4_SOURCE_DIR}/NuttX/*)
+	file(GLOB_RECURSE nuttx_all_files ${PX4_SOURCE_DIR}/NuttX/nuttx/Makefile)
```

Comment:
Whoops, that was temporary. I was trying to generate the build system dependency graph and it was way too big to display. Fixing.